content/dischargeHugoPost.md

```diff
@@ -47,7 +47,7 @@ points(loc$dec_lon_va[1],loc$dec_lat_va[1],
 box(lwd=1.5)
 ```
 
-<img src='/static/ts-colin-discharge/mainPlot-1.png'/ title='Discharge from Tropical Storm Colin'/>
+<img src='/static/ts-colin-discharge/mainPlot-1.png'/ title='Discharge from Tropical Storm Colin' alt='Discharge graph from Tropical Storm Colin' class=''/>
 
 Questions
 =========
```

Comment:
@wdwatkins 
The image below (mainPlot-1.png) is also different using the `xml2` version of dataRetrieval. Can you dig around a bit and try to figure out why?